cmd/aries-agent-mobile/README.md

```diff
@@ -97,10 +112,35 @@ This is an example of how the imported framework can be used:
 /*
 ...
 */
-    ApiAriesController *ac = (ApiAriesController*) AriesagentNewAriesAgent(false, nil);
-    ApiIntroduceController *ic = (ApiIntroduceController*) [ac getIntroduceController:nil];
-    WrappersIntroduceActionsResponse *resp = [ic actions:nil];
-    NSString *actionsResp = resp.actionsResponse;
+    NSError *error = nil;
+
+    // create options
+    ConfigOptions *opts = [[ConfigOptions alloc] init];
+    opts.url = @"http://example.com";
+    opts.useLocalAgent = true;
+    
+    // create an aries agent instance
+    RestAriesREST *ac = (RestAriesREST*) AriesagentNewAriesAgent(opts, &error);
```

Comment:
`AriesagentNewAriesAgent` ?

Reply:
Yeah this is the result of the bindings for iOS. Everything gets prefixed with its package name and it can't be changed. However, Android doesn't have this issue. It's being tracked here: https://github.com/golang/go/issues/32573#issuecomment-650249748